src/prop-types.js

```diff
@@ -25,5 +25,6 @@ export default {
   onChange: React.PropTypes.func,
   skipSuggest: React.PropTypes.func,
   getSuggestLabel: React.PropTypes.func,
-  autoActivateFirstSuggest: React.PropTypes.bool
+  autoActivateFirstSuggest: React.PropTypes.bool,
+  style: React.PropTypes.object
```

Comment:
Define what kind of object as mentioned in the [React docs](https://facebook.github.io/react/docs/reusable-components.html):

```js
// An object taking on a particular shape
optionalObjectWithShape: React.PropTypes.shape({
  color: React.PropTypes.string,
  fontSize: React.PropTypes.number
}),
```